app/filters.tsx

```diff
@@ -73,6 +72,8 @@ export const stateFilters: StateFilter[] = [
     name: 'Finalizing',
     key: 'finalizing',
     when: {
+      resolutionTimestamp: null,
+      currentAnswerTimestamp_gt: nowTimestamp - _24HoursInSeconds,
```

Comment:
I think we could encapsulate this on utils, `oneDayAgoTimestamp` something like this

Reply:
Will create a const here in this file, if it's used elsewhere we should move it to utils